java/java-impl/src/com/intellij/codeInsight/completion/JavaGenerateMemberCompletionContributor.java

```diff
@@ -45,38 +46,84 @@ public static void fillCompletionVariants(CompletionParameters parameters, Compl
     PsiElement position = parameters.getPosition();
     if (psiElement(PsiIdentifier.class).withParents(PsiJavaCodeReferenceElement.class, PsiTypeElement.class, PsiClass.class).
       andNot(JavaKeywordCompletion.AFTER_DOT).accepts(position)) {
+      int priority = -1;
+      boolean followedByWhitespace = psiElement()
+        .withSuperParent(
+          2, // Go to PsiTypeElement
+          psiElement().beforeSiblingSkipping(
+            psiElement(PsiErrorElement.class), // If no annotations are present, the Psi will contain a "identifier expected" node right after
+            psiElement(PsiWhiteSpace.class)
+              //.withText(StandardPatterns.string().matches("(?s) *\n *\n.*")) // Require line to be followed by empty line
```

Comment:
Why is this commented out?

Reply:
It's an idea I experimented I thought you might consider. It's a heuristic trying to differentiate between a user adding a "missing" `@Override` annotation to an existing method vs trying to write a new method/generate an override.

Reply:
I see, thanks! Might be worthy